### PR TITLE
Create checkbox base component #22

### PR DESCRIPTION
**Pull request by andpii**
Examples:
```
-   <app-checkbox text="Sample Text"></app-checkbox>
-   <app-checkbox text="Sample Text" [checked]="false"></app-checkbox>
-   <app-checkbox text="Sample Text" [checked]="true"></app-checkbox>
-   <app-checkbox text="Sample Text" [checked]="false" (stateChanged)="changeState()"></app-checkbox>
-   <app-checkbox text="Sample Text" (stateChanged)="changeState()"></app-checkbox>
```

### CODE REVIEW

File: frontend/src/shared/components/checkbox-component/checkbox.component.sass [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    margin-right: 10.1px
+    background: colorscheme(white-100)
+
+.checkMark
+    opacity: 0
+    position: absolute
+    left: 4.5px
+    top: 5px
+    pointer-events: none
+
```
Comment (ismv) [inline frontend/src/shared/components/checkbox-component/checkbox.component.sass]: fix newline warning pls

File: frontend/src/shared/components/checkbox-component/checkbox.component.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+@Component({
+  selector: 'app-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.sass']
+})
+export class CheckboxComponent implements OnInit {
+  @Input() public Text = '';
```
Comment (ismv) [inline frontend/src/shared/components/checkbox-component/checkbox.component.ts]: use lowercase for fields and variables, please

**Review by ismv — CHANGES_REQUESTED** [review]

File: frontend/src/shared/components/checkbox-component/checkbox.component.sass (hunk 1 above)
Comment (andpii) [reply frontend/src/shared/components/checkbox-component/checkbox.component.sass]: Fixed

File: frontend/src/shared/components/checkbox-component/checkbox.component.ts (hunk 2 above)
Comment (andpii) [reply frontend/src/shared/components/checkbox-component/checkbox.component.ts]: Fixed. Used lowercase

**Review by ismv — APPROVED** [review]
LGTM

**Review by max-voronkin — APPROVED** [review]

**Pull request merged**